docs/tutorial/security.md

```diff
@@ -201,6 +236,10 @@ practice, that means that global objects like `Array.prototype.push` or
 Electron uses the same technology as Chromium's [Content Scripts](https://developer.chrome.com/extensions/content_scripts#execution-environment)
 to enable this behavior.
 
+Even when you use `nodeIntegration: false`, to enforce strong isolation and
+prevent the use of Node primitives, `contextIsolation` must
```

Comment:
_(nit)_ maybe "must also" instead of "must" to underscore the "even when..." intent of the sentence